marabunta/parser.py

```diff
@@ -198,9 +199,14 @@ def _parse_backup(self, version, backup=True, mode=None):
             raise ParseError(u"'backup' key must be a boolean", YAML_EXAMPLE)
         version.backup = backup
 
+    def _parse_i18n_override(self, version, override_translations=False, mode=None):
+        if not isinstance(override_translations, bool):
+            raise ParseError(u"'override_translations' key must be a boolean", YAML_EXAMPLE)
```

Comment:
Checks are failing because the method receives `None` for `override_translations`, and the instance check against `bool` type fails.
Maybe you can normalize the variable before the check like this:
```suggestion
        override_translations = override_translations or False
        if not isinstance(override_translations, bool):
            raise ParseError(u"'override_translations' key must be a boolean", YAML_EXAMPLE)
````
or:
```suggestion
        if override_translations not in (None, True, False):
            raise ParseError(u"'override_translations' key must be a boolean", YAML_EXAMPLE)
```

Reply:
Too soon bro' ^^

Reply:
But comment taken in account

Reply:
Nexyt PR, the good one ^^ https://github.com/camptocamp/marabunta/pull/76